cinema/serializers.py

```diff
@@ -51,13 +62,18 @@ class MovieDetailSerializer(MovieSerializer):
 
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = (
+            "id", "title", "description", "image",
+            "duration", "genres", "actors"
+        )
 
 
 class MovieSessionSerializer(serializers.ModelSerializer):
+    movie_image = serializers.ImageField(source="movie.image", read_only=True)
```

Comment:
movie_image shouldn't be here - this serializer is used for creating a movie session and image shouldnt be passed there